app/src/main/java/com/github/factotum_sdp/factotum/ui/maps/MapsFragment.kt

```diff
@@ -47,11 +59,18 @@ class MapsFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
 
         setMapProperties()
+
     }
 
     private fun setMapProperties() {
```

Comment:
This method is becoming a bit too long and could be refactored into smaller methods.